numpy/lib/tests/test_arraysetops.py

```diff
@@ -99,7 +119,7 @@ def assert_isin_equal(a, b):
 
         #array-likes as both arguments
         c = [(9, 8), (7, 6)]
-        d = (9, 7)
+        d = (9, 7) 
```

Comment:
Not sure why this space was added, but would be better not to

Reply:
uhh I'm not sure either. will fix.